check/tests/pass.rs

```diff
@@ -1,225 +1,265 @@
 extern crate env_logger;
 
-extern crate gluon_base as base;
-extern crate gluon_parser as parser;
-extern crate gluon_check as check;
+use support::base::ast::{self, Expr, Pattern, Typed};
+use support::base::types::{Generic, Kind, Type};
+use support::base::types;
 
-use base::ast;
-use base::ast::Typed;
-use base::types::{Generic, Kind, Type};
-use base::types;
-
-mod functions;
-use functions::*;
+mod support;
 
 macro_rules! assert_pass {
     ($e: expr) => {{
         if !$e.is_ok() {
             panic!("assert_pass: {}", $e.unwrap_err());
         }
-    }}
+    }};
 }
 
 #[test]
 fn function_type_new() {
     let text = r"
 \x -> x
 ";
-    let result = typecheck(text);
+    let result = support::typecheck(text);
+
     assert!(result.unwrap().as_function().is_some());
 }
 
 #[test]
 fn char_literal() {
-    let _ = ::env_logger::init();
+    let _ = env_logger::init();
+
     let text = r"
 'a'
 ";
-    let result = typecheck(text);
-    assert_eq!(result, Ok(Type::char()));
+    let result = support::typecheck(text);
+    let expected = Ok(Type::char());
+
+    assert_eq!(result, expected);
 }
 
 #[test]
 fn byte_literal() {
-    let _ = ::env_logger::init();
+    let _ = env_logger::init();
+
     let text = r"
 1b
 ";
-    let result = typecheck(text);
-    assert_eq!(result, Ok(Type::byte()));
+    let result = support::typecheck(text);
+    let expected = Ok(Type::byte());
+
+    assert_eq!(result, expected);
 }
 
 
 #[test]
 fn function_2_args() {
-    let _ = ::env_logger::init();
+    let _ = env_logger::init();
+
     let text = r"
 \x y -> 1 #Int+ x #Int+ y
 ";
-    let result = typecheck(text);
-    assert_eq!(result,
-               Ok(Type::function(vec![typ("Int"), typ("Int")], typ("Int"))));
+    let result = support::typecheck(text);
+    let expected = Ok(Type::function(vec![support::typ("Int"), support::typ("Int")],
+                                     support::typ("Int")));
```

Comment:
The type construction functions have intentionally short names as they are used everywhere. I don't think it helps readability  to qualify them.